NuclearData/ceNeutronData/ceNeutronCache_mod.f90

```diff
@@ -116,41 +121,30 @@ module ceNeutronCache_mod
   !! Args:
   !!   nMat [in]  -> Number of materials
   !!   nNuc [in]  -> Number of nuclides
-  !!   nMaj [in]  -> Optional. Number of majorant Xss (Default = 1)
```

Comment:
Are we sure we want to get rid of this? Maybe it would have been used for, e.g., holes? I suppose @Mikolaj-A-Kowalski may have some thoughts.

Reply:
I don't have strong feelings, I just thought the code looked neater. Happy to change it back if you both prefer that way.